modules/33acrossAnalyticsAdapter.js

```diff
@@ -390,7 +453,10 @@ function onAuctionInit({ adUnits, auctionId, bidderRequests }) {
 
 function onBidRequested({ auctionId, bids }) {
   for (let { bidder, bidId, transactionId, src } of bids) {
-    getBidsForTransaction(auctionId, transactionId).push({
+    const auction = locals.cache.auctions[auctionId];
+    const adUnits = auction.adUnits.find(adUnit => adUnit.transactionId === transactionId);
```

Comment:
isn't a single adUnit that one returned? maybe the variable name could be `adUnit`